### PR TITLE
Update docs and minor improvements

### PR DESCRIPTION
**Pull request by antoniaklja**
# Changes

- add issue templates
- improve typos in README.md
- add ROADMAP.md

# Submitter Checklist

These are the criteria that every PR should meet, please check them off as you
review them:

- [ ] Includes tests (if functionality changed/added)
- [x] Includes docs (if user facing)
- [x] Commit messages follow [commit message best practices](https://github.com/jenkinsci/kubernetes-operator/blob/master/CONTRIBUTING.md#commit-messages)

_See [the contribution guide](https://github.com/jenkinsci/kubernetes-operator/blob/master/CONTRIBUTING.md) for more details._

## Reviewer Notes

If API changes are included, additive changes must be approved by at least two [OWNERS](https://github.com/jenkinsci/kubernetes-operator/blob/master/OWNERS) and backwards incompatible changes must be approved by [more than 50% of the OWNERS](https://github.com/jenkinsci/kubernetes-operator/blob/master/OWNERS).

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+---
+
+**Describe the solution you'd like**
+A clear and concise description of what you want to happen.
+
+**Describe alternatives you've considered**
+A clear and concise description of any alternative solutions or features you've considered.
+
+**Additional context**
+Add any other context about the feature request here.
```
Comment (SylwiaBrant) [inline .github/ISSUE_TEMPLATE/feature_request.md:17]: Maybe clearly specify in "Additional context" in what cases that would be useful.

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+---
+
+**Describe the bug**
+A clear and concise description of what the bug is.
+
+**To Reproduce**
+Steps to reproduce the behavior.
+
+**Error logs**
+Add error logs about the problem here.
```
Comment (SylwiaBrant) [inline .github/ISSUE_TEMPLATE/bug_report.md:17]: Clearly state to add logs from both jenkins and jenkins-operator. Maybe events?

File: README.md [hunk 3]
```diff
@@ -10,30 +10,29 @@
...
 ## Preliminaries
 
-Considering that this Operator is created for managing instances for Jenkins, 
-it is important to understand what 
+Considering that this operator has been created for managing instances for Jenkins, it is important to understand what 
 - [Jenkins Pipelines](https://jenkins.io/doc/book/pipeline/) and 
 - CasC ([Configuration as Code](https://github.com/jenkinsci/configuration-as-code-plugin)) are.
 
-Jenkins Pipelines use Scripts written in [Groovy](https://groovy-lang.org/) which aid in the CasC aspect of Jenkins.
+Jenkins Pipelines uses scripts written in [Groovy](https://groovy-lang.org/) which aid in the CasC aspect of Jenkins.
```
Comment (SylwiaBrant) [inline README.md:21]: Please revert or use single "Pipeline".

File: README.md [hunk 4]
```diff
@@ -10,30 +10,29 @@
...
 
-Jenkins uses [plugins](https://plugins.jenkins.io/) like CasC to extend it's solution space by carrying out Jobs of different kinds and providing a composable infrastructure for your CI/CD.
+Jenkins uses [plugins](https://plugins.jenkins.io/) like CasC to extend its solution space by carrying out Jobs of different kinds and providing a composable infrastructure for your CI/CD.
```
Comment (SylwiaBrant) [inline README.md:23]: Please revert.

File: ROADMAP.md [hunk 5]
```diff
@@ -0,0 +1,39 @@
...
+
+## Project Vision
+
+With Jenkins Operator project we want to enable our community to run Jenkins in cloud-native environments like Kubernetes, OpenShift and others. Also, support most of the public cloud providers (AWS, Azure, GCP) in terms of additional capabilities like backups, observability and cloud security.  
+
+
+With declarative configuration and full lifecycle management based on Operator Framework this can become the de facto standard for running Jenkins on top of Kubernetes.
+
+
+Now, we have a dedicated team which can bring more engineering effort to the project. We are now justified long term commitment for the OSS project (instead of working on best-effort basis as it was in the past).
```
Comment (SylwiaBrant) [inline ROADMAP.md:13]: "We are now justified long term commitment for the OSS project". I'm not sure I understand. Can you elaborate what message you want to convey?

**Review by SylwiaBrant — CHANGES_REQUESTED** [review]

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 6]
```diff
@@ -0,0 +1,17 @@
+---
```
Comment (tomaszsek) [inline .github/ISSUE_TEMPLATE/bug_report.md:1]: We already have an issue template https://raw.githubusercontent.com/jenkinsci/kubernetes-operator/master/.github/issue_template.md.

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 2 above)
Comment (SylwiaBrant) [reply .github/ISSUE_TEMPLATE/bug_report.md:17]: As far as I recall we were talking about around last 40-50 lines.

File: ROADMAP.md [hunk 7]
```diff
@@ -0,0 +1,39 @@
...
+
+## Technical Roadmap
```
Comment (jakalkhalili) [inline ROADMAP.md:15]: It would be nice to create epics for these tasks.

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 6 above)
Comment (antoniaklja) [reply .github/ISSUE_TEMPLATE/bug_report.md:1]: I wanted to split this into bug and feature request. I hope it makes sense.

**Review by tomaszsek — APPROVED** [review]

**Pull request merged**